content/millennium/r4/request-and-response/communication.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Communication | R4 API
+---
+
+# Communication
+
+* TOC
+{:toc}
+
+## Overview
+
+The Communication resource is a conveyance of information from one entity, a sender, to another entity, a receiver. The information includes encoded data and optionally a related Patient and/or a related Encounter.
+
+The following fields are returned if valued:
+
+* [Communication Id](https://hl7.org/FHIR/communication-definitions.html#Communication.identifier)
+* [In Response To](https://hl7.org/FHIR/communication-definitions.html#Communication.inResponseTo)
+* [Status (Completed)](https://hl7.org/FHIR/communication-definitions.html#Communication.status)
+* [Communication (Category Message, Reminder, Coding Query)](https://hl7.org/FHIR/communication-definitions.html#Communication.category)
+* [Priority](https://hl7.org/FHIR/communication-definitions.html#Communication.priority)
+* [Subject](https://hl7.org/FHIR/communication-definitions.html#Communication.subject)
+* [Topic](https://hl7.org/FHIR/communication-definitions.html#Communication.topic)
+* [Encounter](https://hl7.org/FHIR/communication-definitions.html#Communication.encounter)
+* [Recieved](https://hl7.org/FHIR/communication-definitions.html#Communication.received)
+* [Recipient](https://hl7.org/FHIR/communication-definitions.html#Communication.recipient)
+* [Sent](https://hl7.org/FHIR/communication-definitions.html#Communication.sent)
+* [Sender](https://hl7.org/FHIR/communication-definitions.html#Communication.sender)
+* [Payload](https://hl7.org/FHIR/communication-definitions.html#Communication.payload)
+
+## Terminology Bindings
+
+<%= terminology_table(:communication, :r4) %>
+
+## Extensions
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ID             | Value\[x] Type    | Description
+---------------|-------------------|----------------------------------------------------------------------------------
+`reply-to`     | [`Reference`]     | Used to direct where replies to the communications should be sent. Must be a Reference(Group)
+`email-status` | [`Code`]          | Status of the electronic communication (Pending, Opened, Deleted)
+
+
+## Search
+
+Search for Communications that meet supplied query parameters:
+
+    GET /Communication?:parameters
+
+_Implementation Notes_
+
+* The Content of a communication will be returned through the Binary Resource. A refernece to the payload will be provided in the Payload seciton.
+* All Coding Querys are also Messages. A search for the category of Messages will return Coding Queries as part of the response bundle. A search for the category of Coding Queries will return items that have both the category of Coding Query and Message.
```

Comment:
This seems local-service specific and might not need to be exposed.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/433/commits/3cd1fcea807808fd32a2331bdf6f7fff06e92546

Reply:
Shouldn't "Messages" be swapped with 'notification' with grammar corrected as appropriate?
Should we add the agreed upon SNOMED in parentheses after 'A search for the category of Coding Queries (SNOMED xxx)?